pkg/sql/catalog/tabledesc/structured.go

```diff
@@ -2380,7 +2380,9 @@ func (desc *wrapper) GetStorageParams(spaceBetweenEqual bool) []string {
 			escapedTTLExpirationExpression := lexbase.EscapeSQLString(string(ttl.ExpirationExpr))
 			appendStorageParam(`ttl_expiration_expression`, escapedTTLExpirationExpression)
 		}
-		appendStorageParam(`ttl_job_cron`, fmt.Sprintf(`'%s'`, ttl.DeletionCronOrDefault()))
+		if ttl.DeletionCron != "" {
```

Comment:
If you are only displaying non-defaults now, you can change this to
```
if override := m.DeletionCron; override != "" {
  appendStorageParam(`ttl_job_cron`, fmt.Sprintf(`'%s'`, override))
}
```
because the default case will never be returned by `ttl.DeletionCronOrDefault()`.